scipy/cluster/vq.py

```diff
@@ -314,7 +314,8 @@ def _kmeans(obs, guess, thresh=1e-5):
     return code_book, prev_avg_dists[1]
 
 
-def kmeans(obs, k_or_guess, iter=20, thresh=1e-5, check_finite=True):
+def kmeans(obs, k_or_guess, iter=20, thresh=1e-5, check_finite=True,
+           seed=None):
```

Comment:
Now we advice to mark these as keyword arguments only #13955
```suggestion
def kmeans(obs, k_or_guess, iter=20, thresh=1e-5, check_finite=True,
           *, seed=None):
```